frontend/src/aquifers/components/View.vue

```diff
@@ -616,7 +623,7 @@ export default {
     }
   },
   computed: {
-    ...mapGetters(['userRoles']),
+    ...mapGetters(['userRoles', 'keycloak']),
```

Comment:
It looks like we are already using "userRoles" here so if we can use these instead, we save ourselves adding the keycloak code.